src/module-vsbridge-indexer-agreement/Model/Indexer/Action/Agreement.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright Divante Sp. z o.o.
+ * See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Divante\VsbridgeIndexerAgreement\Model\Indexer\Action;
+
+
+use Divante\VsbridgeIndexerAgreement\Model\ResourceModel\Agreement as AgreementResource;
+
+class Agreement
+{
+    /**
+     * @var AgreementResource
+     */
+    private $resourceModel;
+
+    /**
+     * Agreement constructor.
+     *
+     * @param AgreementResource $agreementResource
+     */
+    public function __construct(
+        AgreementResource $agreementResource
+    ) {
```

Comment:
we can have it one line `public function __construct(        AgreementResource $agreementResource    ) `

Reply:
fixed